internal/parser/parser_test.go

```diff
@@ -3686,6 +3705,78 @@ join(tables:[a,b], on:["t1"], fn: (a,b) => (a["_field"] - b["_field"]) / b["_fie
 				},
 			},
 		},
+		// {
+		// 	name: "unbalanced braces piped into a function",
+		// 	raw:  `map(fn: (r) => { return r._value ) |> count()`,
```

Comment:
Does this one work?

Reply:
Yes. I realized I didn't need that test. I accidentally left it in.